examples/zephyr/lightdb_stream/src/main.c

```diff
@@ -102,48 +103,117 @@ static void temperature_push_handler(struct golioth_client *client,
     return;
 }
 
-static void temperature_push_async(const struct sensor_value *temp)
+static void temperature_push_json(const struct sensor_value *temp, bool async)
 {
     char sbuf[sizeof("{\"temp\":-4294967295.123456}")];
     int err;
 
     snprintk(sbuf, sizeof(sbuf), "{\"temp\":%d.%06d}", temp->val1, abs(temp->val2));
 
-    err = golioth_stream_set_async(client,
-                                   "sensor",
-                                   GOLIOTH_CONTENT_TYPE_JSON,
-                                   sbuf,
-                                   strlen(sbuf),
-                                   temperature_push_handler,
-                                   NULL);
+    if (async)
+    {
+        err = golioth_stream_set_async(client,
+                                       "sensor",
+                                       GOLIOTH_CONTENT_TYPE_JSON,
+                                       sbuf,
+                                       strlen(sbuf),
+                                       temperature_push_handler,
+                                       NULL);
+    }
+    else
+    {
+        err = golioth_stream_set_sync(client,
+                                      "sensor",
+                                      GOLIOTH_CONTENT_TYPE_JSON,
+                                      sbuf,
+                                      strlen(sbuf),
+                                      2);
```

Comment:
@sam-golioth I ended up increasing this timeout from 1s to 2s, and it appears to have fixed [the lightdb_stream test that was failing yesterday due to a timeout](https://github.com/golioth/golioth-firmware-sdk/actions/runs/8913757172/job/24480299831). I've run the tests a couple times and haven't seen this specific failure again, although I haven't been able to get a 100% pass due to other tests randomly failing.

Reply:
That makes sense, the failure you saw was on the 9160dk and we've definitely seen cell network transactions take longer than 1 second in the past.